<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QtCharts::QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_check">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_save">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_clean">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>430</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清除
（只留标准曲线）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_load">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加载标准曲线</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>力矩检测</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="TW_csv">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>201</width>
      <height>601</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="RB_Destructive">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>破坏性测试</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_noDestructive">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>非破坏性测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_allClean">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>全清</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>角度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_angel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="TW_load">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>340</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="PB_clockwise">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>610</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>逆时针1度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_antiCLockwise">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>顺时针1度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>顺时针5度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>610</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>逆时针5度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>压力值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>顺时针0.2度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>逆时针0.2度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_value">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>610</y>
      <width>91</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_N">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>613</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_G">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>637</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_saveImage">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>310</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存图片</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>graphicsView</zorder>
   <zorder>PB_check</zorder>
   <zorder>PB_save</zorder>
   <zorder>PB_clean</zorder>
   <zorder>PB_load</zorder>
   <zorder>label</zorder>
   <zorder>TW_csv</zorder>
   <zorder>RB_Destructive</zorder>
   <zorder>RB_noDestructive</zorder>
   <zorder>PB_allClean</zorder>
   <zorder>TW_load</zorder>
   <zorder>PB_clockwise</zorder>
   <zorder>PB_antiCLockwise</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>LE_value</zorder>
   <zorder>RB_N</zorder>
   <zorder>RB_G</zorder>
   <zorder>PB_saveImage</zorder>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="setup"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>标定</string>
   </property>
   <property name="toolTip">
    <string>标定</string>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="setup">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
